ds3-sdk/src/main/java/com/spectralogic/ds3client/commands/PutObjectRequest.java

```diff
@@ -17,6 +17,7 @@
 
 import com.spectralogic.ds3client.HttpVerb;
 import com.spectralogic.ds3client.models.Checksum;
+import com.spectralogic.ds3client.utils.SeekableByteChannelInputStream;
 
```

Comment:
Because I moved where the file is.  It used to be in the same package as PutObjectRequest, but when I moved it, it now has to be imported.